src/Solnet.Wallet/Utilities/Base58Encoder.cs

```diff
@@ -163,12 +163,12 @@ public override byte[] DecodeData(string encoded)
             // Skip leading zeroes in b256.
             int it2 = size - length;
             // Copy result into output vector.
-            byte[] vch = new byte[zeroes + size - it2];
-            Array.Fill<byte>(vch, 0, 0, zeroes);
+            Span<byte> vch = stackalloc byte[zeroes + size - it2];

```

Comment:
Same thing here, the array is returned, so a new copy is created in the heap. Revert to prevent more CPU usage in the copy and pointless stack allocation.